src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -140,22 +141,22 @@ public void resendStaffNotifications(
         String authorisation = user.getAuthorisation();
 
         Claim claim = claimService.getClaimByReferenceAnonymous(referenceNumber)
-                .orElseThrow(() -> new NotFoundException(String.format(CLAIM_DOES_NOT_EXIST, referenceNumber)));
+            .orElseThrow(claimNotFoundException(referenceNumber));
 
         switch (event) {
```

Comment:
`event` should be an enum rather than a string

Reply:
Have tried this, but it just ends up with pointless boilerplate. It may be more useful if we were more consistent, but Robotics Support (for example) has a separate endpoint for each type and can't reuse the enum. I'm confident the whole support strategy could use an overhaul, as it's grown rather organically and inconsistently over the last 2 years.